pkg/webhook/webhook.go

```diff
@@ -378,7 +378,7 @@ func (wh *WebHook) selfRegistration(webhookConfigName string) error {
 		},
 	}
 
-	mutatingWebhook := v1beta1.Webhook{
+	mutatingWebhook := v1beta1.MutatingWebhook{
```

Comment:
So it has been renamed in a newer version of the k8s API?

Reply:
yes long ago